framework/test/unit/node/consensus/certificate_generation/utils.spec.ts

```diff
@@ -12,13 +12,46 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-// eslint-disable-next-line jest/no-disabled-tests
-describe.skip('utils', () => {
+import { signBLS, generatePrivateKey, getRandomBytes } from '@liskhq/lisk-cryptography';
+import { codec } from '@liskhq/lisk-codec';
+import { Certificate } from '../../../../../src/node/consensus/certificate_generation/types';
+import { MESSAGE_TAG_CERTIFICATE } from '../../../../../src/node/consensus/certificate_generation/constants';
+import { certificateSchema } from '../../../../../src/node/consensus/certificate_generation/schema';
+import { signCertificate } from '../../../../../src/node/consensus/certificate_generation/utils';
+
+describe('utils', () => {
+	const networkIdentifier = Buffer.alloc(0);
+
 	describe('computeCertificateFromBlockHeader', () => {
 		it.todo('');
 	});
+
 	describe('signCertificate', () => {
-		it.todo('');
+		let privateKey: Buffer;
+		let certificate: Certificate;
+		let signature: Buffer;
+
+		beforeEach(() => {
+			privateKey = generatePrivateKey(getRandomBytes(32));
+			certificate = {
+				blockID: Buffer.alloc(0),
+				height: 1000,
+				stateRoot: Buffer.alloc(0),
+				timestamp: 10000,
+				validatorsHash: Buffer.alloc(0),
+			};
+			const encodedCertificate = codec.encode(certificateSchema, certificate);
+			signature = signBLS(
+				MESSAGE_TAG_CERTIFICATE,
+				networkIdentifier,
+				encodedCertificate,
+				privateKey,
+			);
+		});
+
+		it('should sign certificate', () => {
+			expect(signCertificate(privateKey, networkIdentifier, certificate)).toEqual(signature);
```

Comment:
it's better to input the certificate with `signature` and `aggregationBits`, and make sure those are not included in the message